app/Support/Updater.php

```
@@ -1828,44 +1828,29 @@ public function createBackup(): array
-            if (!mkdir($backupDir, 0755, true) && !is_dir($backupDir)) {
-                $this->debugLog('ERROR', 'Impossibile creare directory backup', [
-                    'path' => $backupDir,
-                    'error' => error_get_last()
-                ]);
-                throw new Exception(__('Impossibile creare directory di backup'));
-            }
-
-            // Log the backup start
-            $logId = $this->logUpdateStart($this->getCurrentVersion(), 'backup', $backupDir);
+            $this->debugLog('INFO', 'Inizio backup', ['scope' => $scope]);
+            $result = (new \App\Support\BackupManager($this->db, $this->rootPath))->createBackup($scope);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '1841', column '28'). (undefined)

(MissingImport)